### ISSUE TITLE
Mini map scale bug

### ISSUE THREAD

**Issue by Vadhaka** [tukui-org/ElvUI#1421]
### Troubleshooting Steps

![WoWScrnShot_112424_002614](https://github.com/user-attachments/assets/875c33a7-f3c1-4ce1-a046-931e402c15f6)

### Describe the expected behavior and what actually happened?

I'm setting up the UI after a fresh install of ElvUI after previously deleting the WoW Cache and WTF folders.
I expected the mini map to fit in its frame.  Even after having all other add ons disabled and deleting the Cache and WTF folders, The issue still occurs. I think the problem is between ElvUI and Blizzard's "Edit Mode".  This all started when I removed ElvUI to see if I could make do with Blizzard's UI. I could not. Blizzard's UI still does not have all the functionality of ElvUI so I decided to install ElvUI to use again.

### Reproducing the issue.

I disabled ElvUI in the add on section for all characters because I wanted to give the Blizzard UI a try. In Blizzard's Edit Mode I couldn't get a square map, so I installed the Sexy Map add on. I chose not to go with Blizzard's UI. After removing the Sexy Map add on and reenabling ElvUI, the mini map in ElvUI was no longer properly scaled to fit in its frame. I realized that Sexy Map had left some saved variables behind so I deleted the Cache and WTF folders under the retail WoW folder, and did a fresh install of ElvUI. It seemed to be working fine until I went in to Edit Mode so I could move the Quest Objective frame where I wanted it and the problem resurfaced.

### Lua Errors

```shell
According to Bug Sac there were no errors in my current session.
```

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/main/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by Vadhaka** [comment]
I solved my problem. Apparently it has something to do with creating a custom setting for Blizzard's 'Edit Mode'.
Changing from my custom configuration of Edit Mode to the default selection of Edit Mode fixed the mini map.
I wanted to see what would happen if I switched back to the custom edit mode. When I did, the mini map issue was back.
I suggest just the default Edit Mode setting. However, I did make a new custom one. LOL  So far, no mini map issue. :)
If that changes, I'll update this post.
